src/change_stream.ts

```diff
@@ -538,7 +540,13 @@ export type ChangeStreamEvents<
   end(): void;
   error(error: Error): void;
   change(change: TChange): void;
-} & AbstractCursorEvents;
+  /**
+   * @remarks Note that the `close` event is currently emitted whenever the internal `ChangeStreamCursor`
+   * instance is closed, which can occur multiple times for a given `ChangeStream` instance.
+   * When this event is emitted is subject to change outside of major versions.
+   */
+  close(): void;
```

Comment:
Should we instead emit a new `ChangeStreamCursor:Close` event from the change stream when the underlying cursor is closed?  If we think that's preferable, I suggest that we 1. update the comment only to mention that the current behavior is buggy and 2. file a ticket (that can be done outside of CSOT).

Reply:
We have a ticket filed to address this (NODE-6434), but I can your suggestion to emit the `ChangeStreamCursor:Close` event as a comment for consideration when we eventually go to refine that ticket.

Reply:
Something I was recently considering, we have that ticket set for v7, but wouldn't close have emitted many more times with the recent-ish bug report about close being emitted multiple times from a cursor?

I think we could consider it a bug that it happens more than once, but we can still do the work in that follow up ticket.

Reply:
I don't think we should say "when we emit this event is subject to change inside major versions".  It should always be emitted exactly once, when the change stream itself is closed.  I agree with Neal that if that's now how its working now, that seems like a bug.